staging/src/k8s.io/code-generator/kube_codegen.sh

```diff
@@ -594,6 +599,7 @@ function kube::codegen::gen_client() {
             --output-dir "${out_dir}/${applyconfig_subdir}" \
             --output-pkg "${applyconfig_pkg}" \
             --external-applyconfigurations "${applyconfig_external}" \
+            --openapi-schema "${applyconfig_openapi_schema}" \
```

Comment:
is passing `--openapi-schema ""` valid ?

Reply:
> is passing `--openapi-schema ""` valid ?

That's the default, see here: https://github.com/kubernetes/kubernetes/blob/531726ee654ec318b6f4be53a64700e2577629fe/staging/src/k8s.io/code-generator/cmd/applyconfiguration-gen/args/args.go#L75-L76